### ISSUE TITLE
The sinatra tracer does not record the request id

### ISSUE THREAD

**Issue by jamiehodge** [DataDog/dd-trace-rb#375]
The rack tracer currently adds a request id tag (`X-Request-Id` header) to each span. It would be nice if the sinatra tracer did the same.

**Comment by delner** [comment]
@jamiehodge We're planning on releasing `0.13.0` on Wednesday June 20th, which should include support for this via #427. The request ID should appear as a tag on the Sinatra span by default, but you could further configure which headers you'd like to see via the following:

```ruby
Datadog.configure do |c|
  # Add these request and response headers as tags
  c.use :sinatra, headers: { request: ['Accept-Encoding'], response: ['Content-Encoding'] }
end
```

Will that work for you?

**Comment by delner** [comment]
This should be addressed with our 0.13.0 release. Closing this.
